llvm/lib/Analysis/IR2Vec.cpp

```diff
@@ -297,9 +309,9 @@ StringRef Vocabulary::getVocabKeyForOperandKind(Vocabulary::OperandKind Kind) {
     OPERAND_KINDS
 #undef OPERAND_KIND
   case Vocabulary::OperandKind::MaxOperandKind:
-    llvm_unreachable("Invalid OperandKind");
+    return "UnknownOperand";
   }
-  llvm_unreachable("Unknown OperandKind");
+  return "UnknownOperand";
```

Comment:
Why return unknown rather than make this unreachable/assert? Is it intentional to have things working if there are new operand types and we haven't updated this yet?

Reply:
unreachable has different behavior in release and debug builds. Had faced issues upon using unreachable earlier.

Reply:
Well yeah, it lowers to an assertion in debug mode and it lowers to an optimization hint in release mode. If this is something that should be an assertion, then it needs to be `llvm_unreachable`. If you expect to hit this because you haven't handled cases yet or something, it needs to be a standard return.